script.js

```diff
@@ -24,29 +24,35 @@ function makePageForShows(allShows) {
 
   const wrapper = document.querySelector("#wrapper");
   allShows.forEach(show => {
-    const getImage = show.image !== null ? show.image.medium : "";
+    let {image, id, name, summary, rating, genres, status, runtime, url} = show;
+    const getImage = image !== null ? image.medium : "";
```

Comment:
Putting in a default image if there is no image will help the page look more uniformed without gaps. Select the show bleach on your site to see.